### DISCUSSION TITLE
New features suggestions

### DISCUSSION THREAD

**Discussion by tEctutoRIt** [AnySoftKeyboard/AnySoftKeyboard#2628] [Ideas]
If u could add 
gboard like theme
A clipboard (for saving regularly use things)
Then it would be great!!!
Also thanks for this already amazing app :-D

**Comment by Jonta** [comment]
What do you mean by "gboard like theme"?

Clipboard: https://github.com/AnySoftKeyboard/AnySoftKeyboard/issues/2722

**Comment by tEctutoRIt** [comment]
![Screenshot_20210320-124429](https://user-images.githubusercontent.com/77356742/111862332-b58f1280-897a-11eb-8764-657dfaf936a7.jpg)

**Reply by Jonta** [reply]
That's a screenshot without description. Do you mean the colours, the layout, several glyphs on one key, the top row, a combination of the above?

**Reply by tEctutoRIt** [reply]
Top row and bottom row mainly. But yeah if possible then layout is cool too.

Currently, in anysoft u have to swipe up on spacebar to access the menu, while its so cool in gboard u find it in top row.
